src/chanakya/web/routes.py

```
@@ -144,21 +138,12 @@ async def chat():
         )
 
         used_tools_in_turn = set()
-        if (
-            "intermediate_steps" in response_payload
-            and response_payload["intermediate_steps"]
-        ):
+        if "intermediate_steps" in response_payload and response_payload["intermediate_steps"]:
```

Ruff (0.15.6)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)